releases/release_notes_09_2022.md

```diff
@@ -22,6 +22,7 @@ This release supports the following use cases:
 - Creating a sample COCO workload
 - Creating a COCO workload using a pre-existing encrypted image
 - Creating a COCO workload using a pre-existing encrypted image on hardware with support for Confidential Computing (CC HW)
+  * Intel TDX is fully supported and tested in CI. Currently there are two choices for Intel TDX: QEMU+TDVF and CloudHypervisor + TD-Shim, both can be deployed using operator with `runtimeClassName`: `kata-qemu-tdx` and `kata-clh-tdx`
```

Comment:
Any chance we could move this to the actual use case section?
We want this section to be super short only mentioning the 4 key use cases in a high level

Reply:
I also think that as @fitzthum pointed out the CI doesn't work with the operator so it can't be considered fully supported by the CI (we can add this for the next release in 6 weeks)